fta/samples/utils.py

```diff
@@ -15,8 +16,31 @@ def humansize(nbytes):
     return "%s %s" % (f, suffixes[i])
 
 
+def convert_fathom_sample_to_labeled_sample(fathom_sample, suffix=".html"):
+    soup = BeautifulSoup(fathom_sample, features="html.parser")
+
+    # remove any existing data-fta_id attrs first
+    fta_labeled_elements = [
+        item for item in soup.find_all() if "data-fta_id" in item.attrs
+    ]
+    for element in fta_labeled_elements:
+        del element["data-fta_id"]
```

Comment:
One of the cases we want to handle with the import functionality is when a sample labeled and exported in FTA has been updated by hand. In that case, an existing fta_id could actually point to an instance in the DB. The way this is currently structured, we will update any changes done to the labels by creating new `LabeledElements`, but we'll also leave orphaned `LabeledElements` in the database corresponding to the old `fta_id`. I think we should either remove the orphaned elements or try to update existing ones.

Reply:
I'm not sure either is the best option.  Technically the `LabeledElements` are not orphaned.  The import results in a new `LabeledElement` because a new `Sample` and `LabeledSample` are created.  The original `LabeledSample` will still exist in the database so deleting or updating the orphaned `LabeledElement` will affect the data integrity of the original `LabeledSample`.  

I see this as a bigger issue.  The case you are describing - data is labelled using FTA and exported, then found to have labeling errors which are manually corrected.  The correctly labelled files are then imported into FTA.  My current implementation has the data somewhat duplicated with the only differences being the manual corrections.  Do we instead want to delete the original incorrectly labelled `LabeledSample` and create a new `LabeledSample` using the manually fixed file and using the original Sample id?

If we do leave the original `LabeledSample` in the database (current implementation), it will have to be selectively excluded from a subsequent export.  Deleting the original `LabeledSample` will also keep the LabeledSamples clean for export.

Reply:
Changes have been made to add a superseded_by_id to the `LabeledSample`.  The default filtering will only return `LabeledSamples` that have superseded_by_id=Null and the associated `LabeledElements`. No data is deleted.

If a `LabeledSample` is deleted all associated `LabeledSamples` are deleted.  Manual database steps are required to rollback 

1. delete the specific `LabeledSample` 
2. set the superseded_by_id=Null for the new current record.